test/Error/aggregate_error.js

```diff
@@ -0,0 +1,456 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+   {
+       name: "proptotype constructor",
+       body: function ()
+       {
+            assert.areEqual(AggregateError, AggregateError.prototype.constructor);
+       }
+   },
+   {
+       name: "proptotype errors absent on protptype",
+       body: function ()
+       {
+           assert.isFalse(AggregateError.prototype.hasOwnProperty("errors"));
+       }
+   },
+   {
+       name: "proptotype message",
+       body: function ()
+       {
+           assert.areEqual("", AggregateError.prototype.message)
+       }
+   },
+   {
+        name: "proptotype name",
+        body: function ()
+        {
+            assert.areEqual("AggregateError", AggregateError.prototype.name)
+        }
+    },
+    {
+        name: "prototype type",
+        body: function ()
+        {
+            assert.areEqual("object", typeof AggregateError.prototype);
+        }
+    },
+    {
+        name: "prototype proto",
+        body: function () {
+
+            assert.areEqual(Error.prototype, Object.getPrototypeOf(AggregateError.prototype));
+        }
+    },
+    {
+        name: "name",
+        body: function ()
+        {
+            assert.areEqual("AggregateError", AggregateError.name)
+        }
+    },
+    {
+        name: "length",
+        body: function ()
+        {
+            assert.areEqual(2, AggregateError.length)
+        }
+    },

```

Comment:
Normally in CC tests we do prototype properties as one case with multiple assertions, please can you combine these for neatness/consistency with similar testfiles?